app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -10,7 +10,7 @@ module HostManagedExtensions
         alias_method_chain :set_hostgroup_defaults, :katello_attributes
         alias_method_chain :info, :katello
         alias_method_chain :smart_proxy_ids, :katello
-
+        belongs_to :kickstart_repository, :class_name => "::Katello::Repository", :foreign_key => :kickstart_repository_id
```

Comment:
@jlsherrill I did work on moving to content facet, but pushed it to a diff branch because I was not comfortable testing everything this late in the game.. Look at the diff here https://github.com/parthaa/katello/commit/35b6ead6bf30b02388e9f56eb844b41371d6d57e and tell me what you think..